ci/pipelines/Build.jenkinsfile

```diff
@@ -34,58 +50,98 @@ pipeline
+ 				}
 			}
 		}
 		
 		stage('Build')
 		{	
 			steps
 			{
-				script
+				dir('EncryptionAdapterInterface')
+				{
+					bat script: "conan export-pkg ./conanfile.py ${packageNameInterface}/${version}@systelab/${channel} --profile=${env.WORKSPACE}/${profile} --force",
+						label: "Exporting package ${packageNameInterface}/${version}@systelab/${channel}"
+				}
+				dir('EncryptionAdapterTestUtilities')
 				{
-					def configurations = ['NewtonDebug', 'NewtonRelease', 'SnowDebug', 'SnowRelease']
-					for(int i=0; i < configurations.size(); i++)
+					script
 					{
-						stage('Build ' + configurations[i])
-						{
-							def props = readProperties file:"ci/pipelines/${configurations[i]}.properties"							
-							props.each { propKey, propValue -> println "${propKey}: ${propValue}" }
-
-							sh "conan export-pkg EncryptionAdapterInterface/conanfile.py EncryptionAdapterInterface/${version}@systelab/${channel} -s build_type=${props.conanBuildType} -s compiler.toolset=${props.conanCompilerToolset} -s arch=${props.conanArch} --force"
-							sh "mkdir build-${configurations[i]}"
-							dir("build-${configurations[i]}")
-							{
-								sh "conan install ../EncryptionAdapterTestUtilities -s build_type=${props.conanBuildType} -s compiler.toolset=${props.conanCompilerToolset} -s arch=${props.conanArch} -e VERSION=${version} -e CHANNEL=${channel} -o gtest=${props.conanGTestVersion}"
-								sh "cmake .. -G '${props.cmakeVS}' -A ${props.cmakePlatform}"	
-								sh "cmake --build . --config ${props.cmakeConfiguration}"
-								sh "conan export-pkg ../EncryptionAdapterTestUtilities/conanfile.py EncryptionAdapterTestUtilities/${version}@systelab/${channel} -s build_type=${props.conanBuildType} -s compiler.toolset=${props.conanCompilerToolset} -s arch=${props.conanArch} -o gtest=${props.conanGTestVersion} --force"
-								sh "conan test ../test_package/conanfile.py EncryptionAdapterTestUtilities/${version}@systelab/${channel} -s build_type=${props.conanBuildType} -s compiler.toolset=${props.conanCompilerToolset} -s arch=${props.conanArch} -o gtest=${props.conanGTestVersion}"
+						archs.each
+						{ arch ->
+							configs.each
+							{ config ->
+								stage("Build ${config}|${arch}")
+								{
+									def buildFolder = "build-${config}-${arch}"
+									bat script: "conan install . --install-folder ${buildFolder} --profile=${env.WORKSPACE}/${profile} -e VERSION=${version} -e CHANNEL=${channel} -s build_type=${config} -s arch=${arch}", label: 'Installing dependencies'
+									bat script: "conan build . --build-folder ${buildFolder}", label: 'Building'
+									bat script: "conan export-pkg . ${packageNameTestUtilities}/${version}@systelab/${channel} --profile=${env.WORKSPACE}/${profile} -e VERSION=${version} -e CHANNEL=${channel} -s build_type=${config} -s arch=${arch} --build-folder ${buildFolder} --force", label: "Exporting package ${packageNameTestUtilities}/${version}@systelab/${channel}"
```

Comment:
You don't need to specify the profile, env vars, build_type and arch because you are already providing the build-folder which has it all from the previous install.
So just:
`conan export-pkg . ${packageNameTestUtilities}/${version}@systelab/${channel} --build-folder ${buildFolder} --force`